pkgs/servers/x11/xorg/xwayland.nix

```diff
@@ -1,4 +1,4 @@
-{ stdenv, wayland, wayland-protocols, xorgserver, xkbcomp, xkeyboard_config, epoxy, libxslt, libunwind, makeWrapper, egl-wayland }:
+{ stdenv, wayland, wayland-protocols, xorgserver, xkbcomp, xkeyboard_config, epoxy, libxslt, libunwind, makeWrapper, egl-wayland, useGlobalFontPath ? false }:
```

Comment:
Can you split this on multiple lines: it's better when looking at diffs. Also the options should be visually separate from input packages.

Reply:
Done